src/test-support/fake.ts

```diff
@@ -49,6 +49,7 @@ export type MockEnvKeys = keyof {
   KUMA_ZONEEGRESS_COUNT: string
   KUMA_ZONEINGRESS_COUNT: string
   KUMA_ZONE_COUNT: string
+  KUMA_ZONE_NAME: string
```

Comment:
Ah, awkwardly at the moment, if you want to see items in the list when preview you have to set a dev cookie with the name of the zone you are looking at, e.g. if you are looking at:

```
/gui/zones/zone-cps/port-1/ingresses?page=1&size=50
```

You need to set:

```
KUMA_ZONE_NAME=port-1
```

I didn't think about this too much as once the backend API is there to allow us to filter by zone, we'll have to pass the zone name back to the API/mocks anyway, so once we do that we'll be able to pick up the zone name in the mocks and not have to do this.